test/services/UIBuilderSpec.scala

```diff
@@ -85,6 +86,14 @@ class UIBuilderSpec extends BaseSpec with ProcessJson {
 
     }
 
+    "Include a question hint appended to the question text" in new QuestionTest {
+      uiBuilder.fromStanzaPage(pageWithQuestionHint)(urlMap) match {
+        case s: QuestionPage if s.question.hint.isDefined => succeed
```

Comment:
Could you test the actual contents of the hint here? Would make the test more comprehensive.